package.json

```diff
@@ -3,10 +3,11 @@
   "version": "0.7.0",
   "license": "MIT",
   "scripts": {
-      "type-check": "tsc --noEmit",
-      "build": "npm run build-types && npm run build-js",
-      "build-types": "tsc --emitDeclarationOnly",
-      "build-js": "babel src --out-dir lib --extensions \".ts,.tsx\" --source-maps inline"
+    "type-check": "tsc --noEmit",
+    "type-check:watch": "npm run type-check -- --watch",
+    "build": "npm run build-types && npm run build-js",
+    "build-types": "tsc --emitDeclarationOnly",
```

Comment:
I think this should be `build:types` and `build:js`.

Reply:
@jedmao https://github.com/Microsoft/TypeScript-Babel-Starter/commit/dcd1285c2c9c6fb6afc5729f183decf8adc81c1a @DanielRosenwasser has changed from ':' to '-'

Reply:
Why isn't it `type-check-watch` then? I'm very confused by the inconsistency.

Reply:
@jedmao btw, you always have a chance to make own PR.